### PR TITLE
Update steps.css

### PR DESCRIPTION
**Pull request by Christopher-O**
-removed style for lead zero in striped,-shrunk min height for list and sub list to remove extra spacing when content is two lines or less

### CODE REVIEW

**Comment by duboisp** [comment]
@Christopher-O prior to merge your change, we will need a little bit more such a revised implementation plan and to have tangible evidence that this experimentation are progressing toward a stable component. That evidence can be user research report, accessibility assessment, ACR, translation, proof that DTO completed (or started to draft) the guidance..... as it was specified in the [current implementation plan](https://wet-boew.github.io/GCWeb/m%C3%A9li-m%C3%A9lo/2021-05-steps/d%C3%A9tails-en.html)

I will be there at the [office hours](https://github.com/wet-boew/wet-boew/wiki/WET-Office-hours,-Heures-de-service-de-la-BOEW) to answer are your question.

**Comment by duboisp** [comment]
as discussed,

* [ ] Create a page with the French translation of the working example
* [ ] Produce an accessbility assessment and upload it to the project experimental folder.
* [ ] Seperate the Working example vs the Guidance.
* [ ] Cross-link all the project pages and documents to ensure we can find all the information published via our GCWeb website.

**Comment by Garneauma** [comment]
Pre-approved upon successful review and completion of all work items.

**Comment by Christopher-O** [comment]
Waiting for @duboisp for conversion of WCAG accessibility points to HTML, 
unsure exactly what this task entails
"Cross-link all the project pages and documents to ensure we can find all the information published via our GCWeb website."
@Garneauma could you please advise?

File: méli-mélo/2021-05-steps/steps-doc-en.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    <div class="well">
+      <h3 class="mrgn-tp-0 text-warning h4">Be careful when <span class="glyphicon glyphicon-exclamation-sign"></span></h3>
+      <p>Mismatching the wrong design with the wrong type of content.</p>
+    </div>
+  </section>
+  <section>
+    <h2>Working examples</h2>
+    <ul>
+      <li><a href="2021-05-steps.html">English example</a></li>
+      <li><a href="2021-05-steps-fr.html">French example</a></li>
```
Comment (duboisp) [inline méli-mélo/2021-05-steps/steps-doc-en.html:57]: Broken link

```suggestion
      <li><a href="index.html">English example</a></li>
      <li><a href="index-fr.html">French example</a></li>
```

File: méli-mélo/2021-05-steps/steps-doc-fr.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    <div class="well">
+      <h3 class="mrgn-tp-0 text-warning h4">Soyez prudent lorsque <span class="glyphicon glyphicon-exclamation-sign"></span></h3>
+      <p>Correspondre le mauvais modèle avec le mauvais type de contenu.</p>
+    </div>
+  </section>
+  <section>
+    <h2>Exemples pratiques</h2>
+    <ul>
+      <li><a href="2021-05-steps.html">Exemple anglais</a></li>
+      <li><a href="2021-05-steps-fr.html">Exemple français</a></li>
```
Comment (duboisp) [inline méli-mélo/2021-05-steps/steps-doc-fr.html:57]: Broken links

```suggestion
      <li><a href="index.html">Exemple anglais</a></li>
      <li><a href="index-fr.html">Exemple français</a></li>
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 3]
```diff
@@ -0,0 +1,651 @@
...
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/m%C3%A9li-m%C3%A9lo-demos/2024-09-kejimkujik/2021-05-steps",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Assess the GCWeb meli-melo steps page according to Standard on Web Accessibility (SWA)",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "steps"
```
Comment (duboisp) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:17]: Must be a valid CSS selector, there is no element `<steps>`.

```suggestion
            "@value": ".lst-stps"
```

**Review by duboisp — CHANGES_REQUESTED** [review]
@ouafaaetta please submit your change suggestion for the accessibility report like you did in the other PR. Thanks

I completed the code review with the local testing.  There is a few typo which we will apply prior to merge this change. 

Note: There is some additional change would be required, but those are not mandatory right now like:
* Using jekyll front-matter to specify the other page language
* Adding the 2 HTML placeholder page to render the accessibility assessment in an HTML format.
* Producing the index.json-ld file for documentation by leveraging the documentation provided.

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 4]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:on-input(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:541]:
```suggestion
        {
            "@id": "_:sc_3.2.2",
            "earl:test": "WCAG21:on-input",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 5]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:consistent-navigation(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:553]:
```suggestion
        {
            "@id": "_:sc_3.2.3",
            "earl:test": "WCAG21:consistent-navigation",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 6]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:consistent-identification(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:565]:
```suggestion
        {
            "@id": "_:sc_3.2.4",
            "earl:test": "WCAG21:consistent-identification",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 7]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:error-identification(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:577]:
```suggestion
        {
            "@id": "_:sc_3.3.1",
            "earl:test": "WCAG21:error-identification",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 8]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:labels-or-instructions(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:589]:
```suggestion
        {
            "@id": "_:sc_3.3.2",
            "earl:test": "WCAG21:labels-or-instructions",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 9]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:error-suggestion(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:601]:
```suggestion
        {
            "@id": "_:sc_3.3.3",
            "earl:test": "WCAG21:error-suggestion",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 10]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:error-prevention-legal-financial-data)(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:613]:
```suggestion
        {
            "@id": "_:sc_3.3.4",
            "earl:test": "WCAG21:error-prevention-legal-financial-data",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 11]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:parsing(Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:Pass",
+            "dct:description": "Simple decoration of existing ordered lists",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:625]:
```suggestion
        {
            "@id": "_:sc_4.1.1",
            "earl:test": "WCAG21:parsing",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:Pass",
            "dct:description": "Simple decoration of existing ordered lists",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 12]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:Name, Role, Value (Level A)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        },
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:637]:
```suggestion
        {
            "@id": "_:sc_4.1.2",
            "earl:test": "WCAG21:Name, Role, Value",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        },
```

File: méli-mélo/2021-05-steps/a11y-1-steps.json [hunk 13]
```diff
@@ -0,0 +1,651 @@
...
+            "earl:test": "WCAG21:status-messages(Level AA)",
+            "earl:subject": "_:subject",
+            "earl:outcome": "earl:inapplicable",
+            "dct:description": "As defined",
+            "earl:mode": "earl:semiAuto",
+            "@type": [
+                "earl:TestResult",
+                "earl:Assertion"
+            ]
+        }
```
Comment (ouafaaetta) [inline méli-mélo/2021-05-steps/a11y-1-steps.json:649]:
```suggestion
        {
            "@id": "_:sc_4.1.3",
            "earl:test": "WCAG21:status-messages",
            "earl:subject": "_:subject",
            "earl:outcome": "earl:inapplicable",
            "dct:description": "As defined",
            "earl:mode": "earl:manual",
            "@type": [
                "earl:TestResult",
                "earl:Assertion"
            ]
        }
```

**Review by ouafaaetta — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]
Reviewed and locally tested. Some typos has been also fixed and the implementation plan updated.
This experimental project is now ready for being stabilized in GCWeb. We will perform a full review soon.

**Comment by Christopher-O** [comment]
Changes in the PR

- Reduced excessive spacing for small amount of content
- Removed background color for default list steps number (except in striped mode) as was not showing up properly in high contrast mode
- Updated governance towards stabilization 

Impact for department

- No change from a departmental perspective

Impact for the public

- Created a cleaner look for items using list steps with a smaller amount of content
- Made the numbers more visible for edge case scenarios of print and high contrast

**Comment by Christopher-O** [comment]
A few minor to dos (will create new PR): 

- Fix print issue (some text can be cut off in pattern between pages)
- Try and resolve edge case high contrast mode for Windows FF and MacOS browsers

**Pull request merged**